src/app/zap-templates/zcl/data-model/chip/localization-time-format-cluster.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+
+  <cluster>
+    <domain>General</domain>
+    <name>Localization Time Format</name>
+    <code>0x0038</code>
```

Comment:
Um... This is the same code as the Time Sync cluster.  Looks like a spec bug; please file it?

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/4691